CHANGES.rst

```diff
@@ -573,6 +573,17 @@ Bug Fixes
     will now "set" rather than "add" units to the active set to avoid
     the namespace clash with the default units. [#3873]
 
+  - Three changes were made to the FITS unit format:
+
+    - Only scales of the form ``10^^k``, ``10^k``, ``10+k``, ``10-k``
+      and ``10(k)`` are accepted.
+
+    - Scales that are powers of 10 are written out.  Previously, any
+      non-1.0 scale was rejected.
+
+    - The ``*`` character is accepted as a separator between the scale
+      and the units.
```

Comment:
This not *not* conformant with the standard though. The intent is unambiguous so I don't mind accepting it but this could be pointing out (the standard here is a little ambiguous too, but if you dig in it seems to imply that this is not allowed).